src/Microsoft.FeatureManagement.Telemetry.ApplicationInsights/Microsoft.FeatureManagement.Telemetry.ApplicationInsights.csproj

```diff
@@ -34,6 +34,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.22.0" />
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
```

Comment:
We refers `Microsoft.Extensions.Logging` instead of the abstraction package in `Microsoft.FeatureManagement` project.

Reply:
@zhiyuanliang-ms `Microsoft.Extensions.Hosting.Abstractions` is required here because of the hosted service usage. If the project can reference just the `Abstractions` package, that will generate less dependencies for consumers which is a good thing.

Reply:
I mean, in `Microsoft.FeatureManagement.csproj` we reference`Microsoft.Extensions.Logging` instead of Microsoft.Extensions.Logging.Abstraction`. we can change it to abstraction as well.

Reply:
Ah gotcha. If possible, that would make sense indeed, for the same reason.